api.yaml

```diff
@@ -239,7 +288,7 @@ components:
         - resourceType
         - accountId
         - region
-        - resourceId
+        - arn
```

Comment:
I thought this was removed? Does this repo need to be rebased on master?

Reply:
I branch off of Mikes branch and I don't think he had updated with master. I will see if I can resolve this.